src/ui/componentSection/index.ts

```diff
@@ -28,6 +28,7 @@ interface cardComponentRender {
   props: {
     title?: string,
     subtitle?: string,
-    locationId: string,
+    locationId?: string,
+    afterHeaderLocationId?: string,
```

Comment:
I would rename it to `afterCardContentLocationId` so it is more precise

Reply:
Or what does this property exactly mean?

Reply:
@jleifeld My idea is just to render the an iframe out of the `card__content` because now we only have an `iframe-renderer` inside the `card_content` which is pretty not all the time good UX for some other cases. For example in my case, I would like to have some tabs inside a card, but if I implement it like by default, it will have some padding which is not good looking.

Reply:
I would change this to a more specific API because otherwise the card can look overloaded very fast. Because if several app developer add new tabs at this place we have multiple tab bars in one card. So my suggestion would be that we have a defined API in the card where you can add new tab items. These tab items will be merged together with all other tabs from other apps so that the whole design is unified

Reply:
Should I do something like this? I'll pass the props to the Admin SDK
```js
ui.componentSection.add({
      component: 'card',
      positionId: 'example-product-detail-tab-content',
      props: {
          title: 'Component section example',
          locationId: myLocationId,
          tabs: [
            {
              name: 'test',
              callback: () => {}
            }
          ]
      }
  })
```

And inside Shopware, I will take the props and render to the tabs to the card. What do you think?

Reply:
Yes, this looks really good. Instead of `callback` I would use a location identifier for rendering the content. Otherwise it is great 👍

Reply:
@jleifeld It's getting pretty complicated if I try to render tab content by locationId 😄  Because `signedIframeSrc` will only change if `extension` data changes. How do you think we solve it? 😄